tinygrad/lazy.py

```diff
@@ -203,8 +203,8 @@ def loadop(op, shape:Tuple[sint,...], dtype:DType, device:str, arg=None, src:Opt
 
   # create a constant with the shape and dtype of self
   def const(self, val:Union[float, int]) -> LazyBuffer:
-    # NOTE: dtypes.from_np(self.dtype.np) to deal with image types
-    return LazyBuffer.loadop(LoadOps.CONST, tuple(), dtypes.from_np(self.dtype.np), self.device, arg=val).reshape((1,)*len(self.shape)).expand(self.shape)
+    # NOTE: we force the image dtype to be a float32
```

Comment:
should it be float16 for imageh?

Reply:
no because we never use imageh, see cstyle